scania-digital-design-system/tegel
# build: smoke test

## **Describe pull-request**  
Scheduled install and build for all packages

## **Issue Linking:**  
_Choose one of the following options_
- **Jira:** [CDEP-3483-smoke-test](https://tegel.atlassian.net/browse/CDEP-3483)

## **How to test**  
_Provide detailed steps for testing, including any necessary setup._
1. Go to github actions. Check that it runs and completes when scheduled

## **Checklist before submission**

## **Additional context**  
Might need to change schedule to run more often, for test purposes
